charts/dataspace-connector-bundle/values.yaml

```
@@ -21,107 +21,210 @@
 # This is a YAML-formatted file.
 
 tractusx-connector:
-  # Installation options
+  # @ignore
   install:
-    # -- Deploying a PostgreSQL instance
+    # @ignore
     postgresql: false
-    # -- Deploying a HashiCorp Vault instance
+    # @ignore
     vault: false
 
-  # Participant Configuration
-  participant:
-    id: BPNL00000003AZQP
-
-  # IATP (Identity and Trust Protocol) Configuration
+  # === Bring Your Own ===
   iatp:
+    # -- Decentralized IDentifier (DID) of the connector
+    # @section -- Decentralized Claims Protocol (DCP) Settings (Bring Your Own)
```

please replace everywhere
```suggestion
    # @section -- Decentralized Claims Protocol (DCP, formerly IATP) Settings (Bring Your Own)
```